META6.json

```diff
@@ -5,7 +5,8 @@
   ],
   "build-depends": [
     "PathTools",
-    "JSON::Fast"
+    "JSON::Fast",
+    "Resource::Wrangler"
```

Comment:
Could you make this a full dependency: Resource::Wrangler:ver<1.0.2+>:auth<zef:ab5tract>